index.js

```diff
@@ -587,40 +598,46 @@ function configureBot(bot) {
         bot.whisper(username, 'YES, I will dig - ' + blockName)
       }
 
-      bot.pathfinder.setMovements(defaultMove)
-
       return new Promise(function (resolve, reject) {
         console.log('Moving to block to dig it')
-        bot.pathfinder.goto(new GoalLookAtBlock(theBlock.position, bot.world, {reach: 4}))
-            .then( async () => {
-              const bestHarvestTool = bot.pathfinder.bestHarvestTool(bot.blockAt(theBlock.position))
-              if (bestHarvestTool) {
-                await bot.equip(bestHarvestTool, 'hand')
-              }
-              console.log("Got to the block and the right tool, now to dig it")
-              bot.dig(bot.blockAt(theBlock.position))
-                  .then(() => {
-                    console.log('I dug up a ' + blockName)
-                    if (username) {
-                      bot.whisper(username, 'I dug up a ' + blockName)
-                    }
-                    resolve()
-                  })
-                  .catch(err => {
-                    console.error('ERROR, I had problem trying to dig ' + blockName, err)
-                    if (username) {
-                      bot.whisper(username, 'ERROR, I had problem trying to dig ' + blockName)
+        try {
+          bot.pathfinder.goto(new GoalLookAtBlock(theBlock.position, bot.world, {reach: 4}))
```

Comment:
Same here, this can just be `await bot.pathfinder.goto(...)` and then the next line is what is currently inside?